1.0/docs/devguide/styling.md

```diff
@@ -24,92 +40,148 @@ provided via `<style>` tags placed inside the element's local DOM `<template>`.
             display: block;
             border: 1px solid red;
           }
-          #child-element {
+          /* In shady DOM this will become `.child-element.my-element {` */
+          .child-element {
             background: yellow;
           }
-          /* styling elements distributed to content (via ::content) requires */
-          /* selecting the parent of the <content> element for compatibility with */
-          /* shady DOM . This can be :host or a wrapper element. */
-          .content-wrapper ::content > .special {
-            background: orange;
-          }
+
         </style>
 
-        <div id="child-element">In local DOM!</div>
-        <div class="content-wrapper"><content></content></div>
+        <div class="child-element">In local DOM!</div>
+        <content></content>
 
       </template>
 
       <script>
-
           Polymer({
               is: 'my-element'
           });
-
       </script>
 
     </dom-module>
 
-To place styles outside of the element, or share styles between elements, you can create
-a [style module](#style-modules).
+In the page, you might have:
+
+    <my-element>
+        <!-- In shady DOM the class is "child-element style-scope my-element" -->
+        <div class="child-element">In light DOM!</div>
+    </my-element>
+
+And see that only `In local DOM` has a yellow background.
 
 **Note:**  Prior to Polymer 1.1, the recommendation was to place `<style>` tags
 inside the `<dom-module>` for an element (but _outside_ the `<template>`). This
  is still supported, but is no longer recommended.
 {: .alert .alert-info }
 
+**NOTE: I like introducing what shadow DOM styling SHOULD be like, and then explain what we should do with shady DOM, and why.**
+
+#### Shim limitations
+
+There is no need to do any special workaround in this simple case, since shady
+DOM will automatically change the style as well as the element's class to make
```

Comment:
Run-on sentence. Maybe something like:

Shady DOM emulates shadow DOM style encapsulation by adding classes to elements inside the local DOM tree at runtime, and by rewriting, or _shimming_ the style rules. For example, shady DOM adds the classes `style-scope` and `my-element` to the local DOM tree for `<my-element>`. The selector `.child-element` is rewritten as `.my-element.child-element`.

Shady DOM only shims style rules that appear _inside_ an element's local DOM, or in a [`custom-style` element ](link forward). Styles defined at the document level inside a normal `<style>` tag are **not** shimmed and can leak into local DOM trees. To avoid this issue, use a `custom-style` element for document-level styling.